packages/screens/Rakki/RakkiScreen.tsx

```diff
@@ -0,0 +1,857 @@
+                      fontWeight: "400",
+                    }}
+                  >
+                    {prettyPrice(
+                      networkId,
+                      totalPrice.toString(),
+                      info.config.ticket_price.denom,
+                    )}
+                  </GradientText>
+                </View>
+              </View>
+            </View>
+            <Box
+              notched
+              style={{
+                backgroundColor: "#222",
+                height: 34,
+                paddingHorizontal: 12,
+                marginHorizontal: "auto",
+                justifyContent: "center",
+              }}
+            >
+              <BrandText
+                style={[
+                  fontSemibold13,
+                  {
+                    fontWeight: "400",
+                    color: "#777",
+                    justifyContent: "center",
+                  },
+                ]}
+              >
+                Available Balance{" "}
+                <BrandText
+                  style={[
+                    fontSemibold14,
+                    {
+                      fontWeight: "400",
+                      color: canPay ? "#A3A3A3" : errorColor,
+                    },
+                  ]}
+                >
+                  {prettyPrice(
+                    networkId,
+                    ticketDenomBalance,
+                    info.config.ticket_price.denom,
+                  )}
+                </BrandText>
+              </BrandText>
+            </Box>
+            <View style={{ height: 1, backgroundColor: "#333" }} />
+            <View
+              style={{
+                flexDirection: "row",
+                gap: 20,
+                marginHorizontal: "auto",
+              }}
+            >
+              <SecondaryButton
+                textStyle={{ fontWeight: "400" }}
+                text="Cancel"
+                size="M"
+                onPress={() => setModalVisible(false)}
+              />
+              <PrimaryButton
+                disabled={!canBuy}
+                loader
+                onPress={wrapWithFeedback(async () => {
+                  if (!selectedWallet?.address) {
+                    throw new Error("No wallet with valid address selected");
+                  }
+                  const cosmWasmClient =
+                    await getKeplrSigningCosmWasmClient(networkId);
+                  const feature = getNetworkFeature(
+                    networkId,
+                    NetworkFeature.CosmWasmRakki,
+                  );
+                  if (feature?.type !== NetworkFeature.CosmWasmRakki) {
+                    throw new Error("Rakki not supported on this network");
+                  }
+                  const msgs: MsgExecuteContractEncodeObject[] = [];
+                  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                  for (const _ of range(ticketAmountNumber.toNumber())) {
```

Comment:
If you propose this for optimization, I'd like that you create a benchmark, because doing `i` string to number conversion (due to `i < ticketAmountNumber.toNumber()` being executed on all loop iteration) might actually be slower than the range iteration. optimization is tricky

this is probably faster than both though:
```ts
const len = ticketAmountNumber.toNumber()
for (let i = 0; i < len; i++) {
```

also we really don't need to optimize this IMO

Reply:
It was to remove `// eslint-disable-next-line @typescript-eslint/no-unused-vars`
I understand we need a benchmark as an argument.
Let's leave this comment

Reply:
oh you're right, I didn't think that using a counter would allow to remove the disable, done in 3fca400f0d010dc8c7a50822874bdca1924064b9

sorry I was tired